asf_public_discourse_home_decarbonisation/analysis/mse/eda_mse_category_data.py

```diff
@@ -0,0 +1,311 @@
+"""
+This script performs exploratory data analysis on Money Saving Expert data, by looking at:
+- A few statistics such as: number of posts collected, number of replies to posts and range of dates;
+- Number of posts per year;
+- Distribution of posts per user;
+- Distribution of replies per user;
+- Distribution of interactions per user.
+
+Computed statistics are logged and appear in your terminal.
+Plots are saved in the folder `asf_public_discourse_home_decarbonisation/outputs/figures/mse/`.
+
+To run this script:
+`python asf_public_discourse_home_decarbonisation/analysis/mse/eda_mse_category_data.py`
+
+To change category or date/time range, use the following arguments:
+`python asf_public_discourse_home_decarbonisation/analysis/mse/eda_mse_category_data.py --category <category> --collection_date_time <collection_date_time>`
+where
+<category>: category or sub-forum to be analysed, e.g. "energy", "lpg-heating-oil-solid-other-fuels", etc.
+<collection_date_time>: data collection date/time in the format YYYY_MM_DD
```

Comment:
On that, it would be great to have a list of the different categories at this point, so it's possible to know what you can run over. Guess you'll provide that in the README as well.

Reply:
Information about categories added. README with more information will follow in a different PR, issue created [here](https://github.com/nestauk/asf_public_discourse_home_decarbonisation/issues/12)